src/components/matrixSize.jsx

```diff
@@ -17,10 +17,33 @@ export default class MatrixSize extends React.Component {
       this.setState({ numCols: event.target.value });
   }
   handleClick(){
-  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(parseInt(this.state.numRows));
-  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(parseInt(this.state.numCols));
+  firebase.database().ref('grids/' + this.state.gridId).once('value', gridSnapshot => {
```

Comment:
Yeah I thought about that, and if we want to do it that way we can and it'd be a lot easier haha but I figured they might want to keep their old data when they resize. If you think we should do it the other way though, just let me know